runners/spark/src/main/java/org/apache/beam/runners/spark/metrics/MetricsAccumulatorParam.java

```diff
@@ -18,25 +18,31 @@
 
 package org.apache.beam.runners.spark.metrics;
 
+import org.apache.beam.sdk.metrics.MetricsContainerStepMap;
 import org.apache.spark.AccumulatorParam;
 
 
 /**
  * Metrics accumulator param.
  */
-class MetricsAccumulatorParam implements AccumulatorParam<SparkMetricsContainer> {
+class MetricsAccumulatorParam implements AccumulatorParam<MetricsContainerStepMap> {
```

Comment:
Why is this an issue?

It is instantiated [here](https://github.com/staslev/beam/blob/874c8d0da65568b01cd5f184e303d39c7810a8bf/runners/spark/src/main/java/org/apache/beam/runners/spark/metrics/MetricsAccumulator.java#L63-L63), so instead of `new MetricsAccumulatorParam()` it could be `new MetricsContainerStepMap.AccumulatorParam()`